src/AltinnCore/Designer/Views/Rules/CompilationResult.cshtml

```diff
@@ -1,60 +1,64 @@
 @using AltinnCore.ServiceLibrary.Models;
 
 @{
-    CodeCompilationResult codeResult = ViewBag.CompilationResult;
+  CodeCompilationResult codeResult = ViewBag.CompilationResult;
+    string org = ViewContext.RouteData.Values["org"] as string;
+    string service = ViewContext.RouteData.Values["service"] as string;
 }
 
 @if (codeResult != null && codeResult.CompilationInfo != null)
 {
-    <ul>
-        <li>Errors: @codeResult.Errors</li>
-        <li>Warnings: @codeResult.Warnings</li>
-        <li>TimeUsed: @codeResult.TimeUsed</li>
-    </ul>
+  <ul>
+    <li>Errors: @codeResult.Errors</li>
+    <li>Warnings: @codeResult.Warnings</li>
+    <li>TimeUsed: @codeResult.TimeUsed</li>
+  </ul>
 
-    <table class="table table-striped table-bordered small">
-        <tr>
-            <th>Type</th>
-            <th>Kode</th>
-            <th>File</th>
-            <th>Line</th>
-            <th>Description</th>
-        </tr>
+  <table class="table table-striped table-bordered small">
+    <tr>
+      <th>Type</th>
+      <th>Kode</th>
```

Comment:
Code?

Reply:
I have not done any changes to the code expect added correct routing, this is as it was before